packages/features/ee/workflows/lib/variableTranslations.ts

```diff
@@ -21,14 +21,14 @@ export function getTranslatedText(text: string, language: { locale: string; t: T
     });
 
     variables?.forEach((variable) => {
-      const regex = new RegExp(variable, "g"); // .replaceAll is not available here for some reason
-      const translatedVariable = originalVariables.includes(variable.toLowerCase().concat("_workflow"))
-        ? language.t(variable.toLowerCase().concat("_workflow")).replace(/ /g, "_").toLocaleUpperCase()
-        : originalVariables.includes(variable.toLowerCase().concat("_name_workflow")) //for the old variables names (ORGANIZER_NAME, ATTENDEE_NAME)
-        ? language.t(variable.toLowerCase().concat("_name_workflow")).replace(/ /g, "_").toLocaleUpperCase()
+      const regex = new RegExp(`{${variable}}`, "g"); // .replaceAll is not available here for some reason
```

Comment:
This (+ changes in line 31) fixed a bug in translating some variables in some languages